p2pserver/block_sync.go

```diff
@@ -156,10 +161,26 @@ func (this *BlockSyncMgr) Start() {
 			go this.checkTimeout()
 			go this.sync()
 			go this.saveBlock()
+			height := this.ledger.GetCurrentBlockHeight()
+			if this.emgGovCmd == p2pComm.EmgGovBlkSync &&
+				height >= this.emergencyGovHeight-1 {
+				log.Infof("notify emergency governance sync block %d done, cur height %d",
+					this.emergencyGovHeight-1, height)
+				this.server.notifyEmgGovBlkSyncDone()
+			}
+		case cmd, ok := <-this.emergencyGovCh:
+			if ok {
+				this.onEmergencyGovCmdReceived(cmd)
```

Comment:
should reject the GovCmd if SyncMgr has been in Gov process?

Reply:
No need. Cause the gov status  and commands to sync mgr are under the control of emergency governance module. There won't be negative cmd to SyncMgr.